tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -411,21 +411,20 @@ def test_environment_paths(self):
         """
         try:
             org = make_org(cached=True)
-            payload = {
+            test_env = make_lifecycle_environment({
                 'organization-id': org['id'],
-            }
-            test_env = make_lifecycle_environment(payload)
+            })
         except CLIFactoryError as err:
             self.fail(err)
 
         # Add paths to lifecycle environments
-        result = LifecycleEnvironment.paths({'organization-id': org['id'],
-                                            'permission-type': 'readable'})
-        self.assertEqual(result.return_code, 0,
-                         "return code must be 0, instead got {0}".
-                         format(result.return_code))
-        self.assertEqual(
-            len(result.stderr), 0,
-            "There should not be an error here.")
-        self.assertIn(u'Library >> {0}'.format(test_env['name']),
-                      result.stdout)
+        result = LifecycleEnvironment.paths({
+            'organization-id': org['id'],
+            'permission-type': 'readable',
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0)
```

Comment:
Can you add `result.stderr` as a third parameter to each of these assertions? It'll make this test much easier to debug if it fails.

    self.assertEqual(result.return_code, 0, result.stderr)
    self.assertEqual(len(result.stderr), 0, result.stderr)